src/Display/Homepage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getPosts } from '../redux/slice/postSlice';
+import Movies from '../components/Details';
+import Header from '../components/Header';
+
+import '../index.css';
+
+const Shows = () => {
+  const posts = useSelector((state) => state.post.post);
+  const dispatch = useDispatch();
+
+  const shouldGetPosts = useRef(true);
+  useEffect(() => {
+    if (shouldGetPosts.current) {
+      shouldGetPosts.current = false;
+      dispatch(getPosts());
+    }
+    // eslint-disable-next-line
+    },[]);
+
+  return (
+    <div className="App">
+      <Header />
+
+      <div className="show-container">
+
+        {
+               posts.map((post) => (
+                 <Movies
+                   key={post.id}
+                   title={post.name}
+                   logo={post.logo_100px}
+                   id={post.id}
+                 />
+               ))
+         }
+      </div>
+    </div>
+  );
+};
+
```

Comment:
The home page should have a list of categories that could be filtered using given parameters like movie name or title. However, there is no feature to filter the movies in your app. Kindly check and update it accordingly. :+1: